src/RecoveryServices/RecoveryServices/Vault/GetAzureRMRecoveryServicesVaultSettingsFile.cs

```diff
@@ -86,9 +86,9 @@ public class GetAzureRmRecoveryServicesVaultSettingsFile : RecoveryServicesCmdle
         /// <summary>
         /// Gets or sets Certificate.
         /// </summary>
-        [Parameter(ParameterSetName = ARSParameterSets.ForSiteWithCertificate, Mandatory = true)]
-        [Parameter(ParameterSetName = ARSParameterSets.ByDefaultWithCertificate, Mandatory = true)]
-        [Parameter(ParameterSetName = ARSParameterSets.ForBackupVaultTypeWithCertificate, Mandatory = true)]
+        [Parameter(ParameterSetName = ARSParameterSets.ForSiteWithCertificate, Mandatory = false)]
+        [Parameter(ParameterSetName = ARSParameterSets.ByDefaultWithCertificate, Mandatory = false)]
+        [Parameter(ParameterSetName = ARSParameterSets.ForBackupVaultTypeWithCertificate, Mandatory = false)]
         [ValidateNotNullOrEmpty]
         public string Certificate { get; set; }
```

Comment:
Please follow the standard practice to deprecate a parameter. [Here's an example](https://github.com/Azure/azure-powershell/blob/main/documentation/breaking-changes/breaking-changes-attribute-help.md#a-parameter-is-being-deprecated:~:text=description%20Foo%20bar-,A%20Parameter%20is%20being%20deprecated,-%5BCmdlet)

Reply:
Got it. But we can still make the parameter,  Mandatory = false, as it's not a breaking change ?

Reply:
Correct.

Reply:
Thanks @isra-fel, I will update accordingly.